### ISSUE TITLE
missing some feathers comparing with old plugin "vimfx"

### ISSUE THREAD

**Issue by colben** [ueokande/vim-vixen#167]
### I'm opening this issue because:

- I'll propose some new features

### Description

- copy text in copy mode with a cursor which can move with 'h,j,k,l' and select text we want to copy. In "VimFX", press "c" to show the text area with highlights, choose the near area and move the cursor to the beginning position of the text we want and press "v" to start selection, and then press "y" to copy the selecting text to clipboard.
- copy the location of a link. In "VimFX", press "yf" to show the link with highlights, the location url can be copied to clipboard after we choose its link, press "yy" to copy current url.
- open a link in current tab exactly. In "VimFX", when we choose a link with pressed "f", the link is always opened in current tab, and opened in new tab with pressed "F".
- can vixen work in new tab? I've installed vixen -v0.5 and it doesn't work in new tab. 

#### System configuration

- Operating system: Archlinux x86_64 lastest
- Firefox version: 57
- Vim-Vixen version: 0.5

**Comment by reading123** [comment]
Unfortunately vim vixen cannot work in the about:newtab page. It's a limitation of the api. I am researching ways to get around this, as it's frustrating

> open a link in current tab exactly. In "VimFX", when we choose a link with pressed "f", the link is   always opened in current tab, and opened in new tab with pressed "F".


I'm running same OS, firefox version and Vim Vixen version but I have this functionality.

**Comment by bovine3dom** [comment]
It is possible to override about:newtab (https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/chrome_url_overrides#Syntax) and then run a content script on it, so that you don't lose control.

**Comment by chocolateboy** [comment]
#&#8203;5 (`gi`) is covered by #98.

**Comment by colben** [comment]
yeah, I have noticed the shortcuts keys 'F' and 'f', pressing 'f' is just equal to clicking the link, not forcing to open this link in current tab. There is a website we can test: "https://www.baidu.com"

**Comment by lydell** [comment]
> In "VimFX", press "c" to show the text area with highlights,

I think you mean "v". VimFx never had a "c" shortcut.

**Comment by colben** [comment]
Yes, pressing "v" in VimFX can enter "cursor" mode, the shortcut "c" works in vimperator which means the mode "caret".

**Comment by PedroLopes** [comment]
First and foremost thanks for this awesome extension & great piece of code! Thanks to all Vim-vixen devs!

Secondly, I'd really really second the "copy text mode" feature from VimFX (as @colben pointed out). 

* **How hard is it to get this in vim-vixen?** (I'm not a firefox add-on dev but happy to give it a shot!)

**Comment by PedroLopes** [comment]
p.s.: @colben I cannot reproduce your issue as it seems likeL pressing F will trigger the next open link to open in a new tab, whereas f opens it in the current tab = all the expected behavior. (Firefox quantum + last Vim-Vixen version).
